plugins/generator-1.19.4/forge-1.19.4/procedures/entity_isonrails.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity}.isOnRails())
```

Comment:
1.19.4 doesn't have method `isOnRails` for `Entity`

Reply:
Ok